webroot/src/components/PrivilegePurchaseFinalize/PrivilegePurchaseFinalize.ts

```diff
@@ -230,14 +170,16 @@ export default class PrivilegePurchaseFinalize extends mixins(MixinForm) {
     }
 
     get selectedStatePurchaseDisplayDataList(): Array<object> {
+        const licenseTypeKey = this.licenseTypeSelected;
+
         return this.selectedStatePurchaseDataList.map((state) => {
-            const stateFeeText = `${state?.jurisdiction?.name()} ${this.compactPrivilegeStateFeeText}`;
-            const stateMilitaryPurchaseText = `${state?.jurisdiction?.name()} ${this.militaryDiscountText}`;
+            const stateFeeText = `${state?.jurisdiction?.name()} ${this.$t('payment.compactPrivilegeStateFee')}`;
+            const stateMilitaryPurchaseText = `${state?.jurisdiction?.name()} ${this.$t('military.militaryDiscountText')}`;
             let stateFeeDisplay = '';
             let stateMilitaryDiscountAmountDisplay = '';
 
-            if (state?.fee) {
-                stateFeeDisplay = state.fee.toFixed(2);
+            if (state?.fees?.[licenseTypeKey]) {
+                stateFeeDisplay = state.fees[licenseTypeKey]?.toFixed(2);
```

Comment:
Is `?` necessary given the conditional? 

```suggestion
                stateFeeDisplay = state.fees[licenseTypeKey].toFixed(2);
```

Reply:
oh yep fixed